celery/beat.py

```diff
@@ -97,17 +97,17 @@ def __init__(self, name=None, task=None, last_run_at=None,
         self.kwargs = kwargs
         self.options = options
         self.schedule = maybe_schedule(schedule, relative, app=self.app)
-        self.last_run_at = last_run_at or self._default_now()
+        self.last_run_at = last_run_at or self.default_now()
         self.total_run_count = total_run_count or 0
 
-    def _default_now(self):
+    def default_now(self):
```

Comment:
Since this was a protected method, other people may have already been using it in sub-classes. This was the main reason I suggested that you add a new method or property getter.